File: php/RAM.php
<?php
   
    class RAM{
        //atribut
        private $capacity = 0;
        private $price = 0;

        //konstruktor
        function __construct($capacity = 0, $price = 0){
            $this->capacity = $capacity;
            $this->price = $price;
        }

        //setter & getter
        function setcapacity($capacity){
            $this->capacity = $capacity;
        }
        function getcapacity(){
            return $this->capacity;
        }

        function setprice($price){
            $this->price = $price;
        }
        function getprice(){
            return $this->price;
        }

        function printRAM(){
            echo "RAM         : ".$this->getcapacity(). " GB". "<br/>";
        }

        function __destruct(){  
        }
    } 
?>

File: c++/ram.cpp
class RAM{
    private:
        //atribut
        int capacity, price;

    public:
        //konstruktor
        RAM(){}
        RAM(int c, int p){
            this->capacity = c;
            this->price = p;
        }

        //setter & getter
        void setCapacity(int capacity){
            this->capacity = capacity;
        }
        int getCapacity(){
            return this->capacity;
        }

        void setPrice(int price){
            this->price = price;
        }
        int getPrice(){
            return this->price;
        }

        void printRAM(){
            cout << "RAM         : " << this->capacity << " GB" << endl;
        }

        //destruktor
        ~RAM(){
        }

};
